.github/workflows/build.yml

```diff
@@ -245,7 +245,7 @@ jobs:
         continue-on-error: true
         with:
           tag_name: ${{ steps.release-charts.outputs.tag }}
-          files: report.yaml
+          files: pr-branch/report.yaml
```

Comment:
This is required to publish `report.yaml` as a GitHub release along with the chart tarball.

Reply:
I see